go.mod

```diff
@@ -2,9 +2,5 @@ module github.com/mrparkers/terraform-provider-keycloak
 
 require (
 	github.com/hashicorp/errwrap v1.0.0
-	github.com/hashicorp/go-hclog v0.7.0 // indirect
-	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/hashicorp/hil v0.0.0-20190212132231-97b3a9cdfa93 // indirect
-	github.com/hashicorp/terraform v0.12.0-rc1
-	github.com/mitchellh/go-homedir v1.1.0 // indirect
```

Comment:
are these entries supposed to go away?  I ran `go get github.com/hashicorp/terraform && go mod tidy` against master and these did not disappear for me.

Reply:
Also, `0.12.1` was released today so we may as well update to that version instead.